<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>전략 검증</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbBasicSetting">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>기본설정</string>
    </property>
    <widget class="QLabel" name="txtFinishTime">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종료시간 : </string>
     </property>
    </widget>
    <widget class="QLabel" name="txtStartTime">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>시작시간 : </string>
     </property>
    </widget>
    <widget class="QLabel" name="txtCode">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종목코드 : </string>
     </property>
    </widget>
    <widget class="QLabel" name="txtTimeUnit">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>시간간격 : </string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCode">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>24</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbTimeUnit">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>24</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="txtTimeUnit_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>27</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X 0.001 초</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>23</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>검 색</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dteStart">
     <property name="geometry">
      <rect>
       <x>316</x>
       <y>24</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm:ss</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dteFinish">
     <property name="geometry">
      <rect>
       <x>523</x>
       <y>24</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm:ss</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chRealTime">
     <property name="geometry">
      <rect>
       <x>664</x>
       <y>27</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbControl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>24</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>▶</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>24</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>■</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPause">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>24</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>I I</string>
     </property>
    </widget>
    <widget class="QLabel" name="txtProcessTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>시간 : </string>
     </property>
    </widget>
    <widget class="QSlider" name="hsTimer">
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>25</y>
       <width>531</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dteProcess">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>24</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm:ss</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="twHoga">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>546</width>
      <height>314</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTableWidget" name="twTrade">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>443</y>
      <width>546</width>
      <height>106</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QListWidget" name="lstLog">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>120</y>
      <width>315</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teCommand">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>560</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>설정</string>
    </property>
    <addaction name="actionDBSetting"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDBSetting">
   <property name="text">
    <string>데이터베이스 설정</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
